content/docs/context.md

```diff
@@ -51,6 +95,11 @@ class Message extends React.Component {
   render() {
     return (
       <div>
+        {/*
+          The Message component must take `color` as as prop to pass it to the
+          Button. Using context, the Button could connect to the color context
+          on its own.
+        */} 
```

Comment:
This block comment within JSX does not read very well. Move it into the `render` function as just inline comments?